src/device/wg_controller.rs

```diff
@@ -360,6 +360,7 @@ async fn build_requested_peers_list<
         upgrade_sync,
         proxy_endpoints,
         remote_peer_states,
+        features,
     )
     .await?;
     let mut exit_node_exists = false;
```

Comment:
I would add a `ipv6` check for adding `IpNetwork::V6("::/0".parse()?),` to exit node's allowed ips.

Reply:
@matislovas do you mean on line 375?

Reply:
Oh damn! Missed that :smile:  yeah, lgtm than